apps/rule-manager/app/service/RuleManager.scala

```diff
@@ -80,6 +80,13 @@ object RuleManager extends Loggable {
     }
   }
 
+  def camelToKebab: String => String = _.foldLeft("") {
```

Comment:
This is so we don't have mixed casing like `invalid-externalId`. This isn't an exhaustive translation to kebab-case (e.g. multiple capitals would cause issues), but it does deal with regular camelCase.